include/solvers/petsc_dm_wrapper.h

```diff
@@ -43,6 +49,44 @@ class PetscDMWrapper
 
   ~PetscDMWrapper(){};
 
+private:
+
+  //! Vector of DMs for all grid levels
+  std::vector<std::unique_ptr<DM>> _dms;
```

Comment:
They put "DM" into the global namespace??  :frowning:

Reply:
Sadly yes. And `TS` if/when we add their time solvers.